src/graphql/schemas/LeadSchema.ts

```diff
@@ -9,6 +9,7 @@ const schema = `
   }
 
   input LeadFilters {
+    id: String
```

Comment:
Put both optional, so we are not forced to specify both of them 

id?: String
authorId?: String

Reply:
From the surrounding context, they already are optional. If something is required, it has ! at the end, and if it is optional there is no !, like so: createdAt: String!